python/operator/setup.py

```diff
@@ -1,7 +1,8 @@
+import os
 from setuptools import setup
 
 setup(name='dablchat-operator-bot',
-      version='1.8.5',
+      version=os.environ['DDIT_VERSION'],
```

Comment:
that's nice, so we don't have to manually update the version everywhere

Reply:
Yeah... it also keeps the versions in sync across sub-artifacts, which I think makes more sense given that they're all packaged in the same file.